aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/LossDetector.java

```diff
@@ -151,6 +151,10 @@ private void onGap(final int termId, final UnsafeBuffer buffer, final int offset
     private void activateGap(final int termId, final int termOffset, final int length)
     {
         activeGap.reset(termId, termOffset, length);
+        if (determineNakDelay() == -1)
```

Comment:
Wouldn't it be better to disable NAKs as a static final and just skip the entire activation instead? @mjpt777 thoughts?